app/controllers/spree/taxons_controller_decorator.rb

```diff
@@ -9,24 +10,22 @@ def show
       end
 
       @taxon = Spree::Taxon.friendly.find(params[:id])
-      
       return unless @taxon
+      @searcher = build_searcher(params.merge(taxon: @taxon, include_images: true))
```

Comment:
Please move all this code to load_products function. If there is any problem, then discuss it with me